modules/bedToNrf.cfmod.pl

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env perl
+use warnings;
+use strict;
+use Getopt::Long;
+use FindBin qw($Bin);
+use lib "$FindBin::Bin/../source";
+use CF::Constants;
+use CF::Helpers;
+use File::Copy qw(move);
+
+##########################################################################
+# Copyright 2014, Philip Ewels (phil.ewels@babraham.ac.uk)               #
+#                                                                        #
+# This file is part of Cluster Flow.                                     #
+#                                                                        #
+# Cluster Flow is free software: you can redistribute it and/or modify   #
+# it under the terms of the GNU General Public License as published by   #
+# the Free Software Foundation, either version 3 of the License, or      #
+# (at your option) any later version.                                    #
+#                                                                        #
+# Cluster Flow is distributed in the hope that it will be useful,        #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of         #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          #
+# GNU General Public License for more details.                           #
+#                                                                        #
+# You should have received a copy of the GNU General Public License      #
+# along with Cluster Flow.  If not, see <http://www.gnu.org/licenses/>.  #
+##########################################################################
+
+my %requirements = (
+        'cores'         => '1',
+        'memory'        => '1G',
+        'modules'       => '',
+        'time'          => sub {
+
+                my $cf = $_[0];
+                my $num_files = $cf->{'num_starting_merged_aligned_files'};
+                # Default to 1 if none were parsed for whatever reason
+                $num_files = ($num_files > 0) ? $num_files : 1;
+                # Ususally takes around 5 mins per file. Set to 30 mins just to be sure.
+                return CF::Helpers::minutes_to_timestamp ($num_files * 1 * 30);
+        }
+);
+
+
+# The help text
+my $helptext = "".("-"x15)."\n BedToNrf\n".("-"x15)."\n
+Takes bed file and computes the NRF (non-redundant fraction).
+Uses code taken from https://github.com/mel-astar/mel-ngs/blob/master/mel-chipseq/chipseq-metrics/CalBedNrf.pl
+Output is a txt file with the total number of reads, the number of unique reads and the  NRF. 
+This file is named basename_nrf.txt.\n\n";
+
+# Start your engines...
+my %cf = CF::Helpers::module_start(\%requirements, $helptext);
+
```

Comment:
This is an optional extra, but as this code will now be bundled with Cluster Flow, it could be good to output the Cluster Flow version here?

```perl
warn "---------- BedToNrf version information ----------\n";
warn "Cluster Flow version v".$CF::Constants::CF_VERSION."\n";
warn "\n------- End of BedToNrf version information ------\n";
```